CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
+# Unreleased
+
+- BREAKING CHANGES
+  - Update exports to support ES6 syntax
```

Comment:
this is _a lot_ for one PR. Ideally, I would prefer each of these entries to be a separate PR so that it's easier to understand what's changing and why in the commit.

Reply:
I do agree that it seems like a lot, but it's hard to break this up into smaller PRs because the changes all rely on each other. The description should give a clear overview of what's changing, but I still recommend cloning this branch and looking at the file structure locally. No functionality was changed to `loadScript` and `loadStylesheet` which is essentially the only 2 things this library is doing.